.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Release
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  semrel:
+    name: Semantic Release
+    runs-on: ubuntu-latest
+    permissions:
+      actions: none
+      checks: none
+      contents: none
+      deployments: none
+      issues: none
+      packages: write
```

Comment:
i think it only needs `packages` if it also pushes an image

Reply:
(updated it)